cpp/bench/sg/svc.cu

```diff
@@ -33,7 +33,7 @@ struct SvcParams {
   BlobsParams blobs;
   MLCommon::Matrix::KernelParams kernel;
   ML::SVM::svmParameter svm_param;
-  ML::SVM::svmModel<D> model;
+  ML::SVM::svmModel<D>* model;
```

Comment:
From what I remember, the `svmModel` struct will now store multiple `rmm::device_uvector` instead of simple raw pointers. Because of this, it is preferable not to copy the model. The use of a pointer here prevents the need for a copy.